lib/beaker-hostgenerator/parser.rb

```diff
@@ -226,6 +227,14 @@ def settings_string_to_map(host_settings)
                 "Malformed host settings: #{host_settings}"
         end
       end
+      # Replace the remaining `,` for to make array for arbitrary list if brackets exist
+      settings_pairs.each do |pair|
+        if pair[1].include?("[")
+          pair[1] = pair[1].
+          delete('[]').
+          split(',')
```

Comment:
Since these two lines are a continuation they should be indented over so they're not at the same alignment as the top-level expression